src/pages/Main/Main.jsx

```diff
@@ -19,13 +21,15 @@ const Main = () => {
 
   useEffect(() => {
     axios
-      .get(`http://localhost:3000/data/main-court-list.json`)
-      .then(res => setCourtList(res.data));
-  }, [setCourtList]);
+      .get(`${process.env.REACT_APP_API_URL}/courts?${queries}`)
+      .then(res => {
+        setCourtList(res.data);
+      });
+  }, []);
 
   return (
     <div>
-      {mapPage ? <Map /> : <List />}
+      {mapPage ? <Map courtList={courtList} /> : <List courtList={courtList} />}
 
       <ChangeScreenBuuton onClick={() => changePage()}>
```

Comment:
```suggestion
      <ChangeScreenBuuton onClick={changePage}>
```
인자로 전달할 값이 없다면 이렇게 하셔도 됩니다!